src/PaymentGateways/Gateways/PayPalStandard/Controllers/PayPalStandardWebhook.php

```diff
@@ -28,23 +29,31 @@ public function __construct(WebhookValidator $webhookValidator)
      * Handle PayPal ipn
      *
      * @since 2.19.0
+     * @unreleased Respond with 200 http status to ipn.
      */
     public function handle()
     {
         $eventData = file_get_contents('php://input');
         $eventData = wp_parse_args($eventData);
 
-        if ( ! $this->webhookValidator->verifyEventSignature($eventData)) {
-            wp_die('Forbidden', 404);
+        if (!$this->webhookValidator->verifyEventSignature($eventData)) {
+            exit();
```

Comment:
Why isn't this 403 forbidden anymore? And why aren't we doing a `Log::warning()` here?

Reply:
@JasonTheAdams I tested this on PayPal. For `403` HTTP status on PayPal Ipn, it assumes that there is an error with Ipn URL and it will send an email notification to the PayPal account owner. 

![IPN simulator PayPal Developer 2022-03-04 at 12 52 09 AM](https://user-images.githubusercontent.com/1784821/156637541-79feaba3-353e-4bc6-8939-e14e77ae2f5b.jpg)

For this return, I am responding with a `200` HTTP status. 
PayPal account owner can check Ipn log on PayPal and Website to find issue. 


Let me know what do you think

Reply:
Ah, that makes sense. 👍